infra/modules/database/role_manager/role_manager.py

```diff
@@ -98,6 +98,10 @@ def configure_database(conn: Connection) -> None:
     migrator_username = os.environ.get("MIGRATOR_USER")
     schema_name = os.environ.get("DB_SCHEMA")
 
+    logger.info("Revoking default access on public schema")
+    conn.run("REVOKE CREATE ON SCHEMA public FROM PUBLIC;")
+    conn.run("REVOKE ALL ON DATABASE app FROM PUBLIC;")
+
```

Comment:
as [specified in the ticket](https://github.com/navapbc/template-infra/issues/300) can you move these lines to the configure_schema function

> update [the configure_schema function in role_manager.py](https://github.com/navapbc/template-infra/blob/main/infra/modules/database/role_manager/role_manager.py#L128-L135) to run the commands:

Reply:
actually nvm this is a better spot

Reply:
i realize we broke the ability to connect to the db with this command so i'm gonna need to fix that to unblock platform-test